components/frontend/src/report/ReportDashboard.js

```diff
@@ -78,14 +79,30 @@ export function ReportDashboard({ dates, measurements, onClick, onClickTag, relo
                     key={tag}
                     maxY={nrMetrics}
                     onClick={() => onClickTag(tag)}
+                    selected={anyTagsHidden}
                     summary={summary}
                 />
             )
         })
     }
+    const extraCards = []
+    if (report.issue_tracker?.type && !settings.hiddenCards.includes("issues")) {
+        const selected = settings.metricsToHide.value === "no_issues"
+        extraCards.push(
+            <IssuesCard
+                key="issues"
+                onClick={() => settings.metricsToHide.set(selected ? "none" : "no_issues")}
+                report={report}
+                selected={selected}
+            />,
+        )
+    }
+    if (!settings.hiddenCards.includes("legend")) {
+        extraCards.push(<LegendCard key="legend" />)
+    }
     return (
         <CardDashboard
-            cards={subjectCards.concat(tagCards.concat([<LegendCard key="legend" />]))}
+            cards={subjectCards.concat(tagCards.concat(extraCards))}
```

Comment:
I'm surprised that there is no change needed to the `CardDashboard.cards` prop. It's undesirable to specify more specific typing than `array` ?

Reply:
I've changed the prop type to `arrayOf(Card)`.